src/components/modal/presenter-view/component.tsx

```diff
@@ -83,6 +85,19 @@ export function PresenterViewComponent(props: PresenterViewComponentProps) {
             />
             <span className="check-box-label">Skip Presenter</span>
           </label>
+          <label className="check-box-label-container" htmlFor="skipPickedUsers">
+            <input
+              type="checkbox"
+              id="skipPickedUsers"
+              checked={filterOutPickedUsers}
+              onChange={() => {
+                setFilterOutPickedUsers(!filterOutPickedUsers);
+              }}
+              name="options"
+              value="skipPickedUsers"
+            />
+            <span className="check-box-label">Skip Picked Users</span>
```

Comment:
This name is not really meaningful, I would suggest something like:
"Allow pick same user multiple times" which will be false by default.
Also, there is the need to make necessary changes in the code.

Reply:
"A user may be picked more than once" perhaps?

Reply:
In line with https://github.com/bigbluebutton/plugin-pick-random-user/issues/13, I would suggest "Include already picked users". What do you think?

Reply:
> In line with https://github.com/bigbluebutton/plugin-pick-random-user/issues/13, I would suggest "Include already picked users". What do you think?

That's quite good! @antobinary, what do you think? I'm on board with this one

Reply:
On board too!

Reply:
Done.
But for consistency https://github.com/bigbluebutton/plugin-pick-random-user/issues/13 needs to be adressed too now. Otherwise it looks awkward to have 2x "Skip" options (enabled per default) and then one "Include" (disabled per default).

Reply:
> But for consistency https://github.com/bigbluebutton/plugin-pick-random-user/issues/13 needs to be adressed too now

Don't worry about that, I will handle it, one other thing this PR exposed is that if the list of already picked users is long enough, there is no way to pick another one, because the height of the modal grows indefinitely... I will open an issue for that. And don't worry about that either, the plugin already did that.